### PR TITLE
fix(kind cleanup): fix task SC2015

### PR DESCRIPTION
**Pull request by liu-xg**
#### What type of PR is this?

/kind cleanup

#### What this PR does / why we need it:

cleanup code

#### Which issue(s) this PR fixes:
xref #470 

#### Does this PR introduce a user-facing change?
none

### CODE REVIEW

File: hack/e2e-test.sh [hunk 1]
```diff
@@ -49,7 +49,12 @@ function args() {
         exit 0
         ;;
       --skip | --skip=*)
-        [[ "${arg#*=}" != "${arg}" ]] && SKIPS+=("${arg#*=}") || { SKIPS+=("${2}") && shift; }
+        if [[ "${arg#*=}" != "${arg}" ]]
+        then
+          SKIPS+=("${arg#*=}")
+        else
+          { SKIPS+=("${2}") && shift; }
+        fi
```
Comment (wzshiming) [inline hack/e2e-test.sh:57]:
```suggestion
        [[ "${arg#*=}" != "${arg}" ]] && SKIPS+=("${arg#*=}") || { SKIPS+=("${2}") && shift; } || :
```

Comment (wzshiming) [reply hack/e2e-test.sh:57]: Add the suffix ` || :` for all

Comment (liu-xg) [reply hack/e2e-test.sh:57]: ok

**Comment by wzshiming** [comment]
Do not merge main branches, please rebase and squash

**Review by wzshiming — CHANGES_REQUESTED** [review]
Turn off the SC2015 in the script, then rebase and squash

File: hack/verify-shellcheck.sh [hunk 2]
```diff
@@ -33,7 +33,7 @@ disabled=(
   # this lint prefers command -v to which, they are not the same
   2230
   # TODO:
-  2015
+  #2015
```
Comment (wzshiming) [inline hack/verify-shellcheck.sh:36]:
```suggestion
```

Comment (liu-xg) [reply hack/verify-shellcheck.sh]: ok

**Review by wzshiming — APPROVED** [review]
/approve
/lgtm

**Pull request merged**